lib/requester/util.js

```diff
@@ -116,29 +118,37 @@ module.exports = {
      *
      * @todo: Move this to the SDK.
      * @param request
+     * @param options
      */
-    getRequestBody: function (request) {
-        var options,
+    getRequestBody: function (request, options) {
+        var reqBody,
             mode = _.get(request, 'body.mode'),
             body = _.get(request, 'body'),
             empty = request.body && request.body.isEmpty && request.body.isEmpty(),
```

Comment:
the logic for getting `empty` is weird? you qlredy have `body` on top. and it returns `false` if body is undefined. Is that correct?